CMakeLists.txt

```diff
@@ -122,7 +130,6 @@ if(TRAVIS_CI_BUILD)
 endif()
 
 if(CMAKE_COMPILER_IS_GNUCC)
-  include(CheckCCompilerFlag)
   check_c_compiler_flag(-Og HAS_OG_FLAG)
 else()
   set(HAS_OG_FLAG 0)
```

Comment:
Yes it was intentional, to keep from trigger bad behavior of some other compiler who might have a similar looking flag, but doesn't do the same thing.  Perhaps that's overly conservative, but I have no idea what other kinds of compilers we might need to support, so it seemed the safe thing to do.